src/main/java/org/gradle/profiler/CompositeBuildMutator.java

```diff
@@ -47,7 +49,7 @@ public void afterBuild(BuildContext context, Throwable error) {
 
 	@Override
 	public void afterScenario(ScenarioContext context) {
-		for (BuildMutator mutator : mutators) {
+		for (BuildMutator mutator : Lists.reverse(mutators)) {
```

Comment:
I guess we should also invoke `afterBuild` in reverse, no?

Reply:
Probably not very important, but creating a new list just to iterate through the original backwards seems... unnecessary here.

Reply:
Added Lists.reverse() also to `afterBuild` and `afterCleanup`. Yeah I did not wanted to reverse original list, since I don't want to mutate original list. 

I believe Lists.reverse() does not do an actual copy of a list, but it just creates a wrapper around the list. And looping with indexes is just so 2010 :P

Reply:
Wait, 2010 means old? I thought you were supposed to say "XY is so 1970!" or something. Must be getting old. :)

You're right on `reverse()` being cheap, let's keep it!